.github/workflows/ghp_deploy.yml

```diff
@@ -2,7 +2,7 @@ name: CD
 on:
   push:
     branches:
-      ["main", "master", "271-create-documentation-and-library-presentation"]
+      ["main", "master", "271-create-documentation-and-library-presentation", "fix/fix-inference-tutorial"]
```

Comment:
Please remove all beside `main`, I forgot to remove it. If you want to run is manually you can use GH actions menu and run specific job